### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by caroltaylor** [18F/accessibility#175]
<img width="833" alt="screen shot 2017-01-24 at 6 17 08 pm" src="https://cloud.githubusercontent.com/assets/7704655/22271377/3efcdb98-e262-11e6-93c8-e648b8c46a15.png">
I believe the word should be assistive. Currently, in the accessibility/checklist section the word "assistive" is missing an s "assitive" (See; bullet point A - Critical issues that will cause serious problems and/or stop most users of assitive technology from using the site).

(https://pages.18f.gov/accessibility/checklist/. 

Hope that helps. The site is fantastic.

**Comment by nickbristow** [comment]
Thanks! 
https://github.com/18F/accessibility/commit/02a9ed8e4845c429740bd27c78f08a1d9300519a
